site/confirmdelete.html.jet

```diff
@@ -10,6 +11,7 @@
       <h1>{{ i18n("confirm_delete_header") }}</h1>
     </div>
     <div class="page-content">
+      <p>{{ i18n("confirm_delete_is_permanent") }}</p>
       <p>{{ i18n("confirm_delete_message") }}</p>
```

Comment:
I haven't changed this to a div, when I put it in a div I had to change the css so that the text was formatted.

When adding the div into the css to do the same centering formatting it made the error alert wack since that must be a div with the same principle of you can put some html in it. I feel like I'm running into this because I've done something wrong with the css. 

Could a way around it be making the translation be a html <p> tag. 

I think I remember that you can do rules that directly hit only the children of an element, so that would be the child of the page-content div but that seems like it's potentially a little hacky too. 

Wondering how you guys would go about this...

Reply:
My usual strategy with CSS debugging is to go into devtools, look at the elements that are present and the rules that are affecting it, and to inspect the elements and look at the margins / paddings / size overlays to understand how the boxes are being laid out.

Core template's CSS is a mess and has a bunch of specific rules that can be pretty confusing to work with, as rules with high specificity (a `<p>` that is a direct descendant of `.some-class`) for example would take precedence over a more general rule.

Alignment is tricky too because you could be talking about text alignment (e.g. how inline content flows inside a block), or flexbox alignment (how boxes are positioned inside a flex container).

Also, it's _possible_ that the invalid html `<s72-confirm-delete-form />` is messing with things, although that's more likely to affect elements placed after it.